src/components/Home/LocalWarningCallout.tsx

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-import { useLocalStorage, writeStorage } from '@rehooks/local-storage';
+// import { useLocalStorage, writeStorage } from '@rehooks/local-storage';
```

Comment:
Does this not work at all? If so, please add a TODO comment with link to an issue. Also, I bet we can easily implement this ourselves...

Reply:
Reverted - fixed w/ update